wasi/2021/WASI-05-20.md

```diff
@@ -28,4 +28,7 @@ The meeting will be on a zoom.us video conference.
 1. Announcements
     1. _Sumbit a PR to add your announcement here_
 1. Proposals and discussions
-    1. _Sumbit a PR to add your agenda item here_
+    1. Impact of the canonical ABI proposal on WASI
+       - https://github.com/WebAssembly/interface-types/pull/132
```

Comment:
I am interested in listening in on the discussion. I promise I will be friendly. Would you folks be OK with inviting me?

Reply:
The procedure for asking for an invite is in the agenda.